tests/GetUniqueFirstLettersTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+class GetUniqueFirstLettersTest extends TestCase
+{
+    /**
+     * @dataProvider positiveDataProvider
+     */
+    public function testPositive($input, $expected)
+    {
+        $this->assertEquals($expected, getUniqueFirstLetters($input));
+    }
+
+    public function positiveDataProvider()
+    {
+        return [
```

Comment:
Could you please add more positive test cases and if possible negative test cases.
Also it is good idea to test all other functions.

Reply:
Added additional positive cases.  Implemented negative cases test. Updated code of function to handle exceptions.